site/content/en/docs/for-users/user-guide/workspace.md

```diff
@@ -39,6 +39,6 @@ In addition the workspace also has the following functions:
   - `Shift+C+=`/`Shift+C+-` for contrast.
   - `Shift+S+=`/`Shift+S+-` for saturation.
 
-    ![](../../../../images/image164_mapillary_vistas.jpg)
+    ![](/images/image164_mapillary_vistas.jpg)
```

Comment:
If something starts from `/` it usually means that it is relative to the site root. Probably `images/image164_mapillary_vistas.jpg` should work as well and it is more nature.

Reply:
@nmanovic The "images" folder is at the root of the site. I've added `relativeURLs= true` in `config.toml` so that relative links are generated automatically.